src/test/java/lotto/domain/LottoRankTest.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoRankTest {
+
+    @ParameterizedTest
+    @MethodSource("provideMatchCountAndIsBonusAndLottoRank")
+    @DisplayName("성공 - 당첨 개수에 보너스 볼에 일치하는 로또 등수를 반환한다.")
+    void test(int matchCount, boolean isBonus, LottoRank lottoRank){
+        assertThat(LottoRank.of(matchCount, isBonus)).isEqualTo(lottoRank);
+    }
+
+    private static Stream<Arguments> provideMatchCountAndIsBonusAndLottoRank() {
+        return Stream.of(
+                Arguments.of(3, false, LottoRank.FIFTH),
+                Arguments.of(4, false, LottoRank.FOURTH),
```

Comment:
```suggestion
                Arguments.of(3, false, LottoRank.FIFTH),
                Arguments.of(3, true, LottoRank.FIFTH),
                Arguments.of(4, false, LottoRank.FOURTH),
                Arguments.of(4, true, LottoRank.FOURTH),
```

이 조건으로 테스트 실패하지 않나요?
보너스는 오직 2등일 때에만 영향을 끼쳐야 합니다.